pkg/volume/cinder/attacher.go

```diff
@@ -79,38 +79,32 @@ func (attacher *cinderDiskAttacher) Attach(spec *volume.Spec, hostName string) e
 	if err == nil && attached {
 		// Volume is already attached to node.
 		glog.Infof("Attach operation is successful. volume %q is already attached to node %q.", volumeID, instanceid)
-		return nil
+	} else {
+		_, err = cloud.AttachDisk(instanceid, volumeID)
+		if err != nil {
```

Comment:
So if there is error, we just log it and continue to the next step cloud.GetAttachmentDiskPath?